BeamlineStatusLogger/sources.py

```diff
@@ -194,3 +198,25 @@ def read(self):
             timestamp = datetime.now(self.localtz)
             value = None
         return Data(timestamp, value, metadata=metadata)
+
+
+class TINEAttributeSource:
+    def __init__(self, device_address, attr, metadata={}, tz=None):
+        if tine is None:
+            raise tine_import_err
+        self.device_address = device_address
+        self.attr = attr
+        self.metadata = metadata
+        if tz:
+            self.tzlocal = tz
+        else:
+            self.tzlocal = timezone('Europe/Berlin')
+
+    def read(self):
+        attribut = tine.get(self.device_address, self.attr)
+        test = attribut["data"]
+        value = {self.attr: test}
+        failure = None
```

Comment:
This needs to be set correctly if the `get` call or the `"status"` check failed.